src/engraving/tests/barline_data/barlinedelete-ref.mscx

```diff
@@ -73,115 +75,145 @@
       </Part>
     <Staff id="1">
```

Comment:
But here, still with the old system. Is that desirable?

Reply:
Shouldn't make any difference, but nope thanks! Fixed

Reply:
@cbjeukendrup  Sorry, I was wrong here. The first `<Staff>` tag writes the properties of the Staff itself, so it makes sense to write its EID and linking properties here. The second _<Staff>_ tag instead writes the _content_ of the staff, so we don't need to repeat the EID and link properties. And here the "id" tag still serves as an index, it's basically how the reader knows where it is, so I can't just remove it at the moment. It would be probably quite trivial to do since both the write and the read are sequential, so writing a progressive staff ID is unnecessary. But I'd prefer to do that in a follow up